packages/Babylonian-Core.package/BPExampleTraceInformation.class/instance/trace..st

```diff
@@ -1,6 +1,8 @@
 accessing
 trace: anObject
 	
+	|oldTrace|
+	oldTrace := trace.
 	trace := anObject.
-	self changed: #newTrace with: trace.
+	self changed: #newTrace with: {trace. oldTrace}.
```

Comment:
i need the old trace for the bpprintbuggercontext so it can successfully untrack methods which has been removed in an example's trace. i *could* introduce a new object here but that would be an overhead imo. how to handle this in a "pretty" way?

Reply:
Na, its fine :)